lab_3_generate_by_ngrams/main.py

```diff
@@ -388,6 +562,35 @@ def run(self, prompt: str, seq_len: int) -> Optional[str]:
         In case of corrupt input arguments or methods used return None,
         None is returned
         """
+        if (not isinstance(prompt, str) or not prompt or
+            not isinstance(seq_len, int) or seq_len <= 0):
+            return None
+        prompt_encoded = self._text_processor.encode(prompt)
+        if not prompt_encoded:
+            return None
+        sequence_candidates = {prompt_encoded: 0.0}
+        for prediction in range(seq_len):
+            copy = sequence_candidates.copy()
```

Comment:
do not use built in names for your variables

Reply:
renamed